README.md

```diff
@@ -244,6 +186,7 @@ curl --location --request POST 'http://localhost:8181/v1/data/main' \
 --header 'Content-Type: application/json' \
 --data-raw '{
     "input": {
+        "type": "file",
```

Comment:
this input it missing the active rules

Reply:
Yeah, it's not really needed for this example, missing `activated_rules` mean running all rules.